pinot-controller/src/test/resources/recommenderInput/BooleanInput.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
i dont think we need a dedicated input json for boolean. would you be able to add a field and on additional query in SortedInvertedIndexInput.json?